tests/pytorch_tests/function_tests/test_fully_quantized_exporter.py

```diff
@@ -100,41 +100,28 @@ def test_layers_wrapper(self):
         self.assertTrue(self.fully_quantized_mbv2.features_0_0_bn.is_weights_quantization)
         self.assertFalse(self.fully_quantized_mbv2.features_0_0_bn.is_activation_quantization)
 
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.features_0_2, PytorchQuantizationWrapper))
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.features_0_2.layer, torch.nn.ReLU6))
-        self.assertFalse(self.fully_quantized_mbv2.features_0_2.is_weights_quantization)
-        self.assertTrue(self.fully_quantized_mbv2.features_0_2.is_activation_quantization)
-
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.classifier_0, PytorchQuantizationWrapper))
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.classifier_0.layer, torch.nn.Dropout))
-        self.assertFalse(self.fully_quantized_mbv2.classifier_0.is_weights_quantization)
-        self.assertFalse(self.fully_quantized_mbv2.classifier_0.is_activation_quantization)
```

Comment:
Check that dropout is not quantized

Reply:
how do you do that? the only thing I thought is to check that this member "classifier_0_activation_holder_quantizer" does not exist.